Tests/CMakeOnly/GNUInstallDirs2Other/CMakeLists.txt

```diff
@@ -0,0 +1,75 @@
+cmake_minimum_required(VERSION 3.2.3)
+
+project(GNUInstallDirsTest)
+
+# Set prefix explicitly to the root of the filesystem
+set(CMAKE_INSTALL_PREFIX "/usr/local")
+include(GNUInstallDirs)
+
+set(
+    VAR_LIST
+    BINDIR
+    SBINDIR
+    LIBEXECDIR
+    SYSCONFDIR
+    SHAREDSTATEDIR
+    LOCALSTATEDIR
+    INCLUDEDIR
+    DATAROOTDIR
+    DATADIR
+    INFODIR
+    LOCALEDIR
+    MANDIR
+    DOCDIR
+  )
+
+set(
+    SHORT_DIR_LIST
+    bin
```

Comment:
The expected values could be placed in a table like

    set(expect_short_BINDIR "bin")
    set(expect_long_BINDIR "/usr/local/bin")

and then referenced below as `${expect_short_${dir}}` and `${expect_long_${dir}}`.